src/gui/entry/EntryModel.cpp

```diff
@@ -127,9 +131,29 @@ QVariant EntryModel::data(const QModelIndex& index, int role) const
         return QVariant();
     }
 
+    /**
+     * @author Fonic <https://github.com/fonic>
+     * Copied date/time formatting string from 'EditWidgetProperties.cpp'
+     * TODO: this shouldn't be hardcoded, neither here nor in 'EditWidgetProperties.cpp',
+     *       instead this should be global and user-configurable (or derived automatically
+     *       from current regional settings)
+     */
+    QString timeFormat("d MMM yyyy HH:mm:ss");
```

Comment:
@fonic since this is a constant, this could be declared as such in the `EntryModel` header file.
```cpp
static const QString DefaultTimeFormat("d MMM yyyy HH:mm:ss");
```